src/event.rs

```diff
@@ -118,7 +128,14 @@ pub enum Event {
     /// Failed in sending a message to client, or connection to client is lost
     ClientLost(SocketAddr),
     /// Notify the current list of adult nodes, in case of churning.
-    AdultsChanged(BTreeSet<XorName>),
+    AdultsChanged {
+        /// Existing Adults in our section.
+        existing: BTreeSet<XorName>,
```

Comment:
according to the calculating
```
let existing = old_adults.intersection(&current_adults).copied().collect();
```
naming of `remaining` shall be more accurate. `existing` is little bit mis-leading.